src/iOS/Xamarin.iOS/ViewControllers/SampleInfoViewController.cs

```diff
@@ -0,0 +1,283 @@
+            // Create and configure the views.
+            View = new UIView { BackgroundColor = UIColor.White };
+
+            // Web view for the readme.
+            _readmeView = new WKWebView(new CoreGraphics.CGRect(), new WKWebViewConfiguration());
+            _readmeView.TranslatesAutoresizingMaskIntoConstraints = false;
+
+            // Add navigation delegat for opening readme links in browser.
+            _readmeView.NavigationDelegate = new BrowserLinksNavigationDelegate();
+
+            // View for source code files.
+            _codeView = new UIView { BackgroundColor = UIColor.White, Hidden = true };
+            _codeView.TranslatesAutoresizingMaskIntoConstraints = false;
+
+            // Web view of the source code html.
+            _codeWebView = new WKWebView(new CoreGraphics.CGRect(), new WKWebViewConfiguration());
+            _codeWebView.TranslatesAutoresizingMaskIntoConstraints = false;
+
+            // Button for bringing up alertcontroller to switch between source code files.
+            var toolbar = new UIToolbar();
+            toolbar.TranslatesAutoresizingMaskIntoConstraints = false;
+            _codeButton = new UIBarButtonItem("", UIBarButtonItemStyle.Plain, SourceCodeButtonPressed);
+            toolbar.Items = new UIBarButtonItem[] { _codeButton };
+
+            // Add sub views to code view.
+            _codeView.AddSubviews(_codeWebView, toolbar);
+
+            // Add sub views to main view.
+            View.AddSubviews(_readmeView, _codeView);
+
+            // Lay out the views.
+            NSLayoutConstraint.ActivateConstraints(new[]
+            {
+                 _readmeView.TopAnchor.ConstraintEqualTo(View.TopAnchor),
+                 _readmeView.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                 _readmeView.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+                 _readmeView.BottomAnchor.ConstraintEqualTo(View.SafeAreaLayoutGuide.BottomAnchor),
+
+                 _codeView.TopAnchor.ConstraintEqualTo(View.TopAnchor),
+                 _codeView.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                 _codeView.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+                 _codeView.BottomAnchor.ConstraintEqualTo(View.SafeAreaLayoutGuide.BottomAnchor),
+
+                 _codeWebView.TopAnchor.ConstraintEqualTo(_codeView.TopAnchor),
+                 _codeWebView.LeadingAnchor.ConstraintEqualTo(_codeView.LeadingAnchor),
+                 _codeWebView.TrailingAnchor.ConstraintEqualTo(_codeView.TrailingAnchor),
+                 _codeWebView.BottomAnchor.ConstraintEqualTo(toolbar.TopAnchor),
+
+                 toolbar.TopAnchor.ConstraintEqualTo(_codeWebView.BottomAnchor),
+                 toolbar.LeadingAnchor.ConstraintEqualTo(_codeView.LeadingAnchor),
+                 toolbar.TrailingAnchor.ConstraintEqualTo(_codeView.TrailingAnchor),
+                 toolbar.BottomAnchor.ConstraintEqualTo(_codeView.BottomAnchor),
```

Comment:
toolbar bottom anchor should be constrained to `SafeAreaLayoutGuide.BottomAnchor`; otherwise it looks awkward in landscape.